packages/video_player/video_player_avfoundation/example/macos/Runner.xcodeproj/project.pbxproj

```diff
@@ -110,12 +110,12 @@
 /* End PBXFrameworksBuildPhase section */
 
 /* Begin PBXGroup section */
-		331C80D6294CF71000263BE5 /* RunnerTests */ = {
+		330B3F8E2B1F9C6A00E6DC3F /* RunnerTests */ = {
```

Comment:
The changes in the project file are because RunnerTests was a folder rather than a group, but the folder doesn't actually exist (since the tests are in the `darwin/` shared directory instead) which was causing issues in the Xcode UI.